oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxNetworks.java

```diff
@@ -52,6 +52,7 @@ public static void updateNetworkStats(NetworkIF netIF) {
         netIF.setPacketsRecv(FileUtil.getUnsignedLongFromFile(rxPacketsPath));
         netIF.setOutErrors(FileUtil.getUnsignedLongFromFile(txErrorsPath));
         netIF.setInErrors(FileUtil.getUnsignedLongFromFile(rxErrorsPath));
-        netIF.setSpeed(FileUtil.getUnsignedLongFromFile(speed));
+        long netSpeed = FileUtil.getUnsignedLongFromFile(speed) * 1024 * 1024;
+        netIF.setSpeed(netSpeed < 0 ?0 :netSpeed);
```

Comment:
can you run this through a code formatter?  Wouldn't ask if it was the only change, but I expect some spaces would be added in the next totally unrelated change....